src/surface_stub.jl

```diff
@@ -1,87 +1,106 @@
+"""
+    AbstractSurfaceStub
+
+An abstract type representing a simple stand-in surface model.
+Any concrete type extending this abstract type should have a `cache` field that
+contains the necessary fields for the simulation, at minimum the following:
+    - `T_sfc` (surface temperature [K])
+    - `ρ_sfc` (surface air density [kg / m3])
+    - `z0m` (roughness length for momentum [m])
+    - `z0b` (roughness length for tracers [m])
+    - `beta` (evaporation scaling factor)
+    - `α_direct` (direct albedo)
+    - `α_diffuse` (diffuse albedo)
+    - `area_fraction` (fraction of the grid cell covered by the ocean)
+    - `phase` (phase of the water used to calculate surface humidity)
```

Comment:
This doesn't look accurate. What would this be for land?

Reply:
It's used when we calculate surface humidity [here](https://github.com/CliMA/ClimaCoupler.jl/blob/e049e032dc9bbd122a0a0e7b442db1d8b2d76d90/src/surface_stub.jl#L25-L26). Land has a different [method](https://github.com/CliMA/ClimaCoupler.jl/blob/e049e032dc9bbd122a0a0e7b442db1d8b2d76d90/experiments/ClimaEarth/components/land/climaland_bucket.jl#L238-L239) to compute surface humidity so it doesn't use the phase

Reply:
SurfaceStub has this option so we can distinguish between ocean and ice surfaces using SurfaceStub